src/object/obj_tx.c

```diff
@@ -1171,13 +1171,16 @@ dc_tx_classify_common(struct dc_tx *tx, struct daos_cpd_sub_req *dcsr,
 	struct daos_oclass_attr	*oca;
 	struct daos_cpd_update	*dcu = NULL;
 	struct obj_reasb_req	*reasb_req = NULL;
+	uint64_t		dkey_hash = 0;
```

Comment:
minor, not aligned.

Reply:
clang-format githook will do formatting for you.  It does require a newer version of clang though.